app/app/controls/BaseControl.js

```diff
@@ -137,6 +140,8 @@ export class BaseControl {
 
         this.oldX = dx;
         this.oldY = dy;
+
+        this.isDragging = false;
```

Comment:
It needs to be on mouseup event. Not here? Because when the component will be moved this.isDragging will be false.